tests/clv/test_distributions.py

```diff
@@ -443,3 +444,72 @@ def test_beta_geo_beta_binom_sample_prior(
 
         np.testing.assert_allclose(lt_frequency.mean(), recency.mean(), rtol=0.84)
         np.testing.assert_allclose(lt_recency.mean(), frequency.mean(), rtol=0.84)
+
+
+class TestBGNBD:
+    def test_logp_matches_excel(self):
```

Comment:
Add a docstring citing where in the literature this is referenced.

Also, why aren't any of the other tests used in other distribution blocks copied over?

Reply:
Added